resources/packs/gcp/gcp.lr

```diff
@@ -947,17 +950,17 @@ private gcp.bigquery.routine @defaults("id") {
 }
 
 // GCP Cloud DNS
-gcp.dns {
+private gcp.project.dnsService {
```

Comment:
Can we add an alias to make old queries working?

Reply:
It's there. I added a comment next to the line that does that